atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/OracleDdlConfig.java

```diff
@@ -61,6 +61,13 @@ public boolean enableOracleEnterpriseFeatures() {
 
     @Value.Default
     public boolean enableShrinkOnOracleStandardEdition() {
+        return true;
```

Comment:
I'm guessing there's no reason not to run `SHRINK SPACE COMPACT`? If that's the case then this change should be safe, but maybe should be called out in the release notes.